From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Tue, 2 Jul 2024 22:26:44 +0200
Subject: Default initrdname to initrd.img-${kernel}

The initrd is named `initrd.img-${kernel}` in Debian and Ubuntu.

Forwarded: not-needed
Signed-off-by: Benjamin Drung <benjamin.drung@canonical.com>
---
 dracut-initramfs-restore.sh | 4 ++--
 dracut.sh                   | 2 +-
 man/dracut.conf.5.adoc      | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

--- a/dracut-initramfs-restore.sh
+++ b/dracut-initramfs-restore.sh
@@ -56,8 +56,8 @@
 
 IMG=$(find_initrd_for_kernel_version "$KERNEL_VERSION")
 if [ -z "$IMG" ]; then
-    if [[ -f /boot/initramfs-linux.img ]]; then
-        IMG="/boot/initramfs-linux.img"
+    if [[ -f /boot/initrd.img ]]; then
+        IMG="/boot/initrd.img"
     else
         echo "No initramfs image found to restore!"
         exit 1
--- a/dracut.sh
+++ b/dracut.sh
@@ -1172,7 +1172,7 @@
         outfile="$dracutsysrootdir$efidir/Linux/linux-$kernel${MACHINE_ID:+-${MACHINE_ID}}${BUILD_ID:+-${BUILD_ID}}.efi"
     else
         if ! [[ $initrdname ]]; then
-            initrdname="initramfs-${kernel}.img"
+            initrdname="initrd.img-${kernel}"
         fi
         if [[ -d "$dracutsysrootdir"/efi/loader/entries || -L "$dracutsysrootdir"/efi/loader/entries ]] \
             && [[ $MACHINE_ID ]] \
--- a/man/dracut.conf.5.adoc
+++ b/man/dracut.conf.5.adoc
@@ -350,7 +350,7 @@
 _initr*$\{kernel\}*_ file pattern and only one file with this pattern should
 exists in the directory where initramfs is loaded from.
 +
-Defaults to _initramfs-$\{kernel\}.img_.
+Defaults to _initrd.img-$\{kernel\}_.
 
 == Files
 _/etc/dracut.conf_::
